client/components/Forms/SubmitForm.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class SubmitForm extends Component {
+  formSubmitHandler = (e) => {
+    e.preventDefault();
+    this.props.submitForm();
+  }
+
+  render() {
+    const { formText } = this.props;
+    return (
+      <form
+        id="delete-event-form"
+        className="delete-form"
+        onSubmit={this.formSubmitHandler}
+      >
+        <div style={{fontSize: '20px'}}>{formText}</div>
+      </form>
+    );
+  }
+}
+
+SubmitForm.defaultProps = { formText: 'Authenticate Andela Socials to have access to your Andela Calendar' };
```

Comment:
I am thinking we should leave out this default value since we intend on making this component reusable and this text is a highly custom default